nixos/modules/virtualisation/keystone.nix

```diff
@@ -0,0 +1,101 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+let
+  cfg = config.virtualisation.keystone;
+  keystoneConf = pkgs.writeText "keystone.conf" ''
+    [DEFAULT]
+    admin_token = SuperSecreteKeystoneToken  # TODO: generate
+
```

Comment:
I think this might be a security issue, especially if keystone.conf is world-readable.

Keystone must be "bootstrapped":
1) use admin_token with a secret token
2) create an "admin" user"
3) remove "admin_token" from the configuration
4) use Keystone through the admin user